build/lib/android/index.js

```diff
@@ -2,46 +2,39 @@
 
 const path = require('path');
 const fs = require('fs-extra');
-const AndroidSDK = require('./androidsdk');
-const ant = require('./ant');
 const git = require('../git');
```

Comment:
<!--
  0 failure: 
  1 warning:  build/lib/android...
  
  
  DangerID: danger-id-Danger;
  File: build/lib/android/index.js;
  Line: 5;
-->

- :warning: build/lib/android/index.js line 5 – 'git' is assigned a value but never used. (no-unused-vars)